### ISSUE TITLE
listen eaddrinuse 3000

### ISSUE THREAD

**Issue by steveblaydes** [phonegap/phonegap-app-desktop#531]
So Im using the Phonegap desktop and whenever I try to open my App I get this message: listen eaddrinuse 3000. Only problem is the port is not use. Ive made sure its allowed through any firewall and whenever I change the port Ill get the same message for the specific port. Any ideas please?

**Comment by oodavid** [comment]
Ditto. The command line tells me that PhoneGap is using the port...

````
lsof -n -i:3000 | grep LISTEN
PhoneGap  556 david   46u  IPv6 0x4084e19a98371265      0t0  TCP *:hbci (LISTEN)
````

**Comment by hermwong** [comment]
We have a potential fix for this issue with #518 which will be included in the PhoneGap Desktop 0.2.2 release.

**Comment by steveblaydes** [comment]
I'm seeing they at least get one project to open but get the port in use when trying to open a second. I cant open anything at all. I will try reverting back to an older version.

**Comment by hermwong** [comment]
Could you guys let me know if you're using both the PhoneGap CLI and PhoneGap Desktop or if you're using the PhoneGap Desktop on its own?

**Comment by steveblaydes** [comment]
I am using the phonegap desktop only.

**Comment by steveblaydes** [comment]
Though I did also install the cll and also told me port was in use

**Comment by hermwong** [comment]
Trying to troubleshoot the issue, could you provide the following info:

Does this happen every time that you launch the PhoneGap Desktop app?
Have you confirmed that you do not have any other apps or processes that are using the same port?ou 
Have you tried using another port?

**Comment by steveblaydes** [comment]
Yes this is every time. I haven't been able to get this to work yet. Yes I turned off any firewall that may block that port even made an exception for it just in case and task manager shows its not in use. I have changed the port several times to other ports I know are open and got the same issue.

**Comment by steveblaydes** [comment]
Thank you for looking into it. I appreciate it.

**Comment by hermwong** [comment]
We have a potential fix for this issue. It'll be patched in our 0.2.2 release.

**Comment by hermwong** [comment]
See #537 for commit history related to the port in use issue.
